pkgs/by-name/he/heimdall/package.nix

```diff
@@ -0,0 +1,71 @@
+{
+  lib,
+  stdenv,
+  fetchFromSourcehut,
+  cmake,
+  kdePackages,
+  libusb1,
+  pkg-config,
+  zlib,
+  enableGUI ? false,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "heimdall${lib.optionalString enableGUI "-gui"}";
+  version = "2.2.1";
+
+  src = fetchFromSourcehut {
+    owner = "~grimler";
+    repo = "Heimdall";
+    rev = "v${version}";
+    sha256 = "sha256-x+mDTT+oUJ4ffZOmn+UDk3+YE5IevXM8jSxLKhGxXSM=";
+  };
+
+  buildInputs =
+    [
+      libusb1
+      zlib
+    ]
+    ++ lib.lists.optionals enableGUI [
+      kdePackages.qtbase
+    ];
+  nativeBuildInputs =
+    [
+      cmake
+      pkg-config
+    ]
+    ++ lib.lists.optionals enableGUI [
+      kdePackages.wrapQtAppsHook
+    ];
+
+  cmakeFlags = [
```

Comment:
Please use `lib.cmakeBool` and `lib.cmakeFeature`

Reply:
Not sure why this doesn't show as "outdated", but it has been done (and the other two change requests have been handled as well).